README.md

```diff
@@ -108,6 +108,10 @@ Add codacycoverage to your `.travis.yml`:
 ```yml
 # .travis.yml
 language: php
+
+env:
+  - CODACY_PROJECT_TOKEN=<yourTokenGoesHere>
```

Comment:
This is needs to be private. In circle ci they have a way to add it in the UI in travis they must have something similar.